worldengine/cli/main.py

```diff
@@ -381,7 +393,7 @@ def main():
     sys.setrecursionlimit(args.recursion_limit)
 
     if args.number_of_plates < 1 or args.number_of_plates > 100:
-        usage(error="Number of plates should be in [1, 100]")
+        usage(error="Number of plates should be a in [1, 100]")
```

Comment:
Did you fork WorldEngine? If so, being up-to-date with "master" might mean that you are up-to-date with *your* fork, not the actual master.
(This again is something I recently changed.)